src/vllm_tgis_adapter/http.py

```diff
@@ -51,7 +51,7 @@ async def run_http_server(
     **uvicorn_kwargs,  # noqa: ANN003
 ) -> None:
     # modified copy of vllm.entrypoints.openai.api_server.run_server that
-    # allows passing of the engine
```

Comment:
The upstream method also allows passing the engine

Reply:
That does not seem to be the case: https://github.com/vllm-project/vllm/blob/main/vllm/entrypoints/openai/api_server.py#L351-L372